docker/latest/docker-compose-ipld-eth-server.yml

```diff
@@ -2,22 +2,26 @@ version: "3.2"
 services:
   ipld-eth-server:
     restart: unless-stopped
+    depends_on:
+      - ipld-eth-db
     image: vulcanize/ipld-eth-server:v2.0.0
     environment:
       IPLD_SERVER_GRAPHQL: "true"
       IPLD_POSTGRAPHILEPATH: http://graphql:5000
       ETH_SERVER_HTTPPATH: 0.0.0.0:8081
       VDB_COMMAND: "serve"
       ETH_CHAIN_CONFIG: "/tmp/chain.json"
-      DATABASE_NAME: "vulcanize_testing"
-      DATABASE_HOSTNAME: "ipld-eth-db"
+      DATABASE_NAME: vulcanize_testing
+      DATABASE_HOSTNAME: ipld-eth-db
       DATABASE_PORT: 5432
       DATABASE_USER: "vdbm"
       DATABASE_PASSWORD: "password"
       ETH_CHAIN_ID: 99
       ETH_FORWARD_ETH_CALLS: $eth_forward_eth_calls
       ETH_PROXY_ON_ERROR: $eth_proxy_on_error
       ETH_HTTP_PATH: $eth_http_path
+      WATCHED_ADDRESS_GAP_FILLER_ENABLED: $watched_address_gap_filler_enabled
+      WATCHED_ADDRESS_GAP_FILLER_INTERVAL: $watched_address_gap_filler_interval
```

Comment:
`WATCHED_ADDRESS_GAP_FILLER_ENABLED` and `WATCHED_ADDRESS_GAP_FILLER_INTERVAL` are not required in v3.2.2.
Watched addresses fill service in previous [version](https://github.com/vulcanize/ipld-eth-server/releases/tag/v3.2.0)